smoke-tests/features/pageobjects/form-designer.page.js

```diff
@@ -54,8 +74,15 @@ class FormDesignerPage extends Page {
 
   getFormPageText() {
     this.formPages.filter((page) => {
-      console.log(page.getText());
+      return console.log(page.getText());
     });
   }
+
+  getNumberInArray(term) {
+    return this.formPageTitles.filter((value) => {
+      return value.getText() === term;
+    }).length;
+  }
 }
+
 module.exports = new FormDesignerPage();
```

Comment:
I would be careful about initialising your classes here. iirc, if you import a module, it will be cached. you'll end up using the same instance of your class/es each time. is this what you intended?

Reply:
@jenbutongit this is the intended behaviour in this case however, at some point in the future I would like to discuss better ways of initialisation.